tests/utils.py

```diff
@@ -46,6 +46,8 @@
 USE_POSTGRES_FOR_TESTS = os.environ.get("SYNAPSE_POSTGRES", False)
 LEAVE_DB = os.environ.get("SYNAPSE_LEAVE_DB", False)
 POSTGRES_USER = os.environ.get("SYNAPSE_POSTGRES_USER", "postgres")
+POSTGRES_HOST = os.environ.get("SYNAPSE_POSTGRES_HOST", "localhost")
```

Comment:
Hang on, can we keep this defaulted to None? (And `SYNAPSE_POSTGRES_USER` too?) That way it will just use the same default connection method as normal `psql`, rather than having different defaults.

(For me, if the postgres user and host are both None then the connection just works, whereas with these defaults it doesn't)

Reply:
done